config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -18,7 +17,7 @@ presubmits:
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250815-171060767f-master
+      - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20250815-171060767f-master
```

Comment:
was just checking for https://github.com/kubernetes/test-infra/blob/99226b10c0f9794477215348e4a4c444e22472ca/config/prow/autobump-config/prow-job-autobump-config.yaml#L41

Reply:
We use it widely now, I want to roll it our for more core jobs.